src/main/scala/cromwell/engine/backend/package.scala

```diff
@@ -3,6 +3,11 @@ package cromwell.engine
 import scala.concurrent.{ExecutionContext, Future}
 
 package object backend {
+  // Ordered by shards, and then ordered by attempts
+  type AttemptedCallLogs = Seq[Seq[CallLogs]]
```

Comment:
It's mainly Seq which is the problem child since it sits waaaaay up there in the hierarchy.